packages/vite/src/node/config.ts

```diff
@@ -606,6 +606,15 @@ export interface ResolvedConfig
       ssr: ResolvedSSROptions
       assetsInclude: (file: string) => boolean
       logger: Logger
+      /**
+       * createResolver is deprecated. It only works for the client and ssr
+       * environments. The `aliasOnly` option is also not being used any more
+       * Plugins should move to createIdResolver(environment.config) instead.
+       * create an internal resolver to be used in special scenarios, e.g.
+       * optimizer & handling css @imports
+       *
+       * @deprecated Use `createIdResolver` instead.
+       */
```

Comment:
```suggestion
      /**
       * Create an internal resolver to be used in special scenarios, e.g.
       * optimizer & handling css `@imports`.
       *
       * This API is deprecated. It only works for the client and ssr
       * environments. The `aliasOnly` option is also not being used any more.
       * Plugins should move to `createIdResolver(environment.config)` instead.
       *
       * @deprecated Use `createIdResolver` from `vite` instead.
       */
```

Updated the docs a bit so it describes what the function does first, and added some missing grammar. Though the main change is updating the `@deprecated` comment to mention importing from `vite` so it's not mistaken to be at `resolvedConfig.createIdResolver` or something.